.ci/packaging.groovy

```diff
@@ -154,6 +155,31 @@ pipeline {
                   prepareE2ETestForPackage("${BEATS_FOLDER}")
                 }
               }
+              stage('Package Docker images for linux/arm64'){
+                agent { label 'arm' }
+                options { skipDefaultCheckout() }
+                when {
+                  beforeAgent true
+                  expression {
+                    return params.linux
+                  }
+                }
+                environment {
+                  HOME = "${env.WORKSPACE}"
+                  PACKAGES = "docker"
+                  PLATFORMS = [
+                    'linux/arm64',
+                  ].join(' ')
+                }
+                steps {
+                  withGithubNotify(context: "Packaging Linux ${BEATS_FOLDER}") {
+                    deleteDir()
+                    release()
+                    pushCIDockerImages()
+                  }
+                  prepareE2ETestForPackage("${BEATS_FOLDER}")
```

Comment:
Mmm we are not supporting the E2E tests for ARM yet. Once this PR is merged and we are pushing ARM-based images to our registries, then we can enable them again. We have to do some preparation in the E2E side, like selecting the target platform based on how they are called. If you all agree, we can remove this line, and we will contribute it back once ready.
```suggestion
```